.github/workflows/docker.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
We do not create releases . This should be on [push.tags](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags)